lib/pages/people/service/person_provider.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:acs_upb_mobile/pages/people/model/person.dart';
+import 'package:acs_upb_mobile/pages/people/view/person_view.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter/material.dart';
+
+extension PersonExtension on Person {
+  static Person fromSnap(DocumentSnapshot snap) {
+    return Person(
+      name: snap.data['name'],
+      email: snap.data['email'],
+      phone: snap.data['phone'],
+      office: snap.data['office'],
+      position: snap.data['position'],
+      photo: snap.data['photo'],
+    );
+  }
+}
+
+class ListPage extends StatefulWidget {
+  @override
+  _ListPageState createState() => _ListPageState();
+}
+
+class _ListPageState extends State<ListPage> {
+  Future _data;
```

Comment:
Give it a better name. It's very unclear what this data represents (e.g. you can call it `people`. You also don't need to make it private (the leading underscore) because the state itself is private anyway.